scripts/bootstrap.sh

```diff
@@ -13,59 +13,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Enter repository root
-cd "$( dirname "${BASH_SOURCE[0]}" )"/..
+
+# Repository root
+ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"/.. || exit
 
 # Load symlink script
-source scripts/symlink.sh
+source "$ROOT"/scripts/symlink.sh
 
 # Clone mbed-os
-if [ -d "dependencies/mbed-os" ]
+if [ -d "$ROOT/dependencies/mbed-os" ]
 then
     echo "Using existing mbed-os"
 else
     # git clone https://github.com/ARMmbed/mbed-os.git
     # Use feature branch until merged to master
-    git clone --depth 1 https://github.com/paul-szczepanek-arm/mbed-os.git -b unittest dependencies/mbed-os
+    git clone --depth 1 https://github.com/ARMmbed/mbed-os.git -b feature_unittest_refactor "$ROOT"/dependencies/mbed-os
 fi
 
 # Add symlinks
-symlink dependencies/mbed-os       tests/UNITTESTS/mbed-os
-
-symlink dependencies/mbed-os       tests/TESTS/LinkLoss/device/mbed-os
-symlink services/LinkLoss          tests/TESTS/LinkLoss/device/LinkLoss
+symlink "$ROOT"/dependencies/mbed-os "$ROOT"/tests/UNITTESTS/mbed-os
 
-symlink dependencies/mbed-os       tests/TESTS/DeviceInformation/device/mbed-os
-symlink services/DeviceInformation tests/TESTS/DeviceInformation/device/DeviceInformation
+symlink "$ROOT"/dependencies/mbed-os "$ROOT"/tests/TESTS/FOTA/device/mbed-os
+symlink "$ROOT"/common               "$ROOT"/tests/TESTS/FOTA/device/common
+symlink "$ROOT"/descriptors          "$ROOT"/tests/TESTS/FOTA/device/descriptors
+symlink "$ROOT"/services/FOTA        "$ROOT"/tests/TESTS/FOTA/device/FOTA
 
-# Create mbed-os.lib for CMake builds
-echo "https://github.com/ARMmbed/mbed-os" > tests/TESTS/LinkLoss/device/mbed-os.lib
-echo "https://github.com/ARMmbed/mbed-os" > tests/TESTS/DeviceInformation/device/mbed-os.lib
```

Comment:
Mbed CLI 2 requires that a `mbed-os.lib` be present in the project root.

The following is also required for the FOTA service:

```shell
echo "https://github.com/ARMmbed/mbed-os" > tests/TESTS/FOTA/device/mbed-os.lib
```